src/Uno.UI/UI/Xaml/Controls/VisualStatesImplementation.tt

```diff
@@ -153,11 +176,16 @@ namespace <#= mixin.NamespaceName #>
 
 		private void UpdateCommonStates(bool useTransitions)
 		{
-			var combinedState = GetCombinedStatePrefix() + GetCombinedStateSuffix();
-            if (combinedState.IsNullOrEmpty()) {
-                combinedState = __normalState;
-            }
-			VisualStateManager.GoToState(this, combinedState, useTransitions);
+#if <#= mixin.HasCommonCheckedState #> || <#= mixin.HasCombinedCheckedState #>
+			var state = GetFullState();
+#else
+			var state = GetCommonState();
+#endif
+//			var combinedState = GetCombinedStatePrefix() + GetCombinedStateSuffix();
```

Comment:
Not used anymore ?

Reply:
No the `combinedState` is now computed by the `GetFullState()` or `GetCommonState()`, will remove it.